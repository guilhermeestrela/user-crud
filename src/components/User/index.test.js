import { render } from '@testing-library/react'
import '@testing-library/jest-dom'

import { User } from './'

describe('User', () => {
    it('Should render User correctly', () => {
        const userMock = {
            id: 1637098364443,
            name: "Test User",
        }
        const { container, getByText } = render(<User user={userMock} />)

        expect(container).toMatchSnapshot()
        expect(getByText('Test User')).toBeInTheDocument()
    })
})
