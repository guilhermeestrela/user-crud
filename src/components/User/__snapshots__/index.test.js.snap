// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`User Should render User correctly 1`] = `
<div>
  <li
    class="user-item"
  >
    <p
      class="user-name"
    >
      Test User
    </p>
    <button
      class="secondary-button"
    >
      Remove
    </button>
  </li>
</div>
`;
